package.json

```diff
@@ -27,6 +27,7 @@
   "devDependencies": {
     "@testing-library/jest-dom": "^6.4.2",
     "@testing-library/react": "^14.2.2",
+    "@testing-library/user-event": "^14.5.2",
```

Comment:
why? I don't see you using it here

Reply:
Again some suggestion somewhere while trying to solve issues with tests. Removed it and tests don't fail so it's good.